joomla/joomla-cms
# [4.0] Help scroll top

Pull Request for Issue # .

Summary of Changes

Added 'onclick' => 'window.scroll(0,0)' to Administrator Joomla! Help page

Testing Instructions

Go to Administrator / Help / Start Here panel / Joomla! Help
Scroll down to the bottom of the list and click an item there

Actual result BEFORE applying this Pull Request

Nothing appears to happen because the Iframe with the Help page has changed out of site.

Expected result AFTER applying this Pull Request

The page scrolls to the top and the new Help page is seen

Documentation Changes Required

None
